src/Plotting/plot_recipes.jl

```diff
@@ -82,6 +83,15 @@ julia> plot(Bs, 1e-2)  # faster but less accurate than the previous call
             if Xi isa Intersection
                 res = plot_recipe(Xi, ε, Nφ)
             else
+                if !isbounded(Xi)
+                    # bound unbounded set with the plotting range
+                    p = plot!()
+                    xlim = xlims(p)
+                    ylim = ylims(p)
+                    low = [xlim[1], ylim[1]]
+                    high = [xlim[2], ylim[2]]
+                    Xi = intersection(Xi, Hyperrectangle(low=low, high=high))
+                end
```

Comment:
Just a note: This part comes from my attempt but does not work (used for plotting lists of sets, which I never tested in the end).

Reply:
Thanks, I will remove it.

Reply:
You might have to add some other code here instead (as I said, I never tested it).